### ISSUE TITLE
Gather all examples from blog posts

### ISSUE THREAD

**Issue by EmiOnGit** [bevyengine/bevy-website#591]
Repost of [this issue](https://github.com/bevyengine/bevy/issues/7935) since it belongs here instead of the bevy repo :)

## What problem does this solve or what need does it fill?
First of, I'm only writing about the 0.10 and up coming releases, everything else is unrealistic
Me and I think a lot of other users often remember a cool example from the blog posts they want to reproduce/study.
Currently, there is almost no way to find the source besides asking around.
I think we all are aware that making a new release is stressful enough and no one has the time nor the extra motivation to collect and document these examples.
However, I would argue that a big part of the problem is that no infrastructure exists to collect the examples.

## What solution would you like?
A list of all examples with links to there repositories or creators.
It should be clear to the reader that the examples were not written for educational reason but to show this specific feature,
therefore the code quality doesn't have to be perfect. If someone wants to take a look it is on their own risk.

This list could be inside a small external repository which the blog post links to.
Alternatively, it could also be another subside of the website similar to the migration guide

## What alternative(s) have you considered?
To let the external repository contain the examples.
Would have the advantage that the example owner couldn't just change or delete his example
How ever I like the thought that some time after the release the example owner could still cleanup the example if wanted.

Additional context
I also think it would be a nice way to honor the people doing the examples.

**Comment by alice-i-cecile** [comment]
During initial discussions, the big thing here is to avoid inducing additional maintenance burden. The examples should work on the version for the release notes, but don't need to be kept updated.

**Comment by EmiOnGit** [comment]
Agreed, it should be made clear that this is only meant as a resource for people that are interested and *not* some kind of official example provided by bevy.

**Comment by IceSentry** [comment]
Personally, I would really like having a bevy-release-samples repo with a new folder for every release and I would prefer if rules around merging and reviewing were relaxed for that repo. Like maintainers being allowed to merge without multi approvals.

**Comment by EmiOnGit** [comment]
Alright, seems like the idea positively received.

Would you want to include the 0.10 blog post or wait for the next release?
Also, would have in mind that *every* showcase picture/video would get a reference or just the more interesting ones?
(pictures from examples or previous versions excluded).

I've quickly made a list with all people(7) that commited a picture/video (not filtered) to the post and could ask around if the code was even saved.

I think we could however at least wait till tomorrow with the asking since we have the release to celebrate :D

**Comment by alice-i-cecile** [comment]
I'm totally fine to operate this on a best effort basis. Might as well include what we can from 0.10 too :)

**Comment by IceSentry** [comment]
Yeah, a couple of people have already asked me about my examples for the prepass and I'd like to share the code more officially.

**Comment by cart** [comment]
I wouldn't mind a samples repo maintained at a best-effort / with reduced scrutiny. Especially if the samples are allowed to "fall behind" bevy versions. We can do that under the `bevyengine` org umbrella.

**Comment by cart** [comment]
(provided we're still meticulous about licensing and attribution)

**Comment by EmiOnGit** [comment]
Alright, since I'm not to deeply integrated in the bevy cult I'm unsure how to advance with this.
I think if we decide to open a sample repo @cart would be the one to create the repo.
As it is created I could
*  start with the structuring (as a new pr)
*  write to the people responsible for the images/videos
Not sure I understood you correctly @alice-i-cecile (I'm not native)

> I'm totally fine to operate this on a best effort basis.

Did you recommend to include sources to *all* media?

**Comment by alice-i-cecile** [comment]
Yep, that's a great summary of the steps needed and by whom. Cart's off for a couple days taking a replacement weekend, but I'll remind him when he's back if needed :)

**Comment by cart** [comment]
If yall can agree on a name I can create it. Do we want to limit it to "blog post samples" or expand it to accommodate anything?

**Comment by IceSentry** [comment]
I think this should only be for blog post samples, I'm not really sure what else could live there. Maybe have more cleaned up, official samples of larger projects that aren't just examples, but I think that should live in a separate place.

**Comment by alice-i-cecile** [comment]
Yeah, I would just go with `bevy_blog_samples`. Doesn't need to be fancy.

**Comment by IceSentry** [comment]
I'd prefer using `-` to be consistent with other repos under bevyengine/

**Comment by mockersf** [comment]
Something to think about before putting on a git repo is the assets size. The forest ruing scenes is 161MB, the model and animations for animation transitions is 23MB.

Even GitLFS is not possible with those kind of volumes, so we should find a way to host assets elsewhere
